examples/users.ts

```diff
@@ -46,7 +48,17 @@ mySdk.users.UserProfile(
   })
 
 mySdk.users.CreateToken(
-  { identity: '<userEmail>', secret: '<password>', domain_id: '<domainId>' }
+  { email: '<email>', secret: '<password>' }
```

Comment:
when I try to create a token with email I get an error:
![image](https://github.com/user-attachments/assets/73956564-e0f5-4738-a9de-e6829fcf75de)